<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Referenciel_Geographique</class>
 <widget class="QDockWidget" name="Referenciel_Geographique">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Référentiel Géographique</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="HL_topbt">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pb_help">
        <property name="text">
         <string>Aide</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/plugins/Geonature-GIS-main/icons/help.png</normaloff>:/plugins/Geonature-GIS-main/icons/help.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hspacertop">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_reset">
        <property name="text">
         <string>Réinitialisation</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/plugins/Geonature-GIS-main/icons/refresh.png</normaloff>:/plugins/Geonature-GIS-main/icons/refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="FL_filter">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_typezonage">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>A preset to fetch.</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(5, 144, 110);</string>
        </property>
        <property name="text">
         <string>Sélection du zonage* :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_source">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(5, 144, 110);</string>
        </property>
        <property name="text">
         <string>Sources :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QgsCheckableComboBox" name="ccb_source">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QListWidget" name="lw_zonage">
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pb_test_selection">
        <property name="text">
         <string>Test sélections (zonages et sources) -&gt; résultat dans console Python QGis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QgsCollapsibleGroupBox" name="cgb_advancedparam">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Paramètres avancés</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QFormLayout" name="FL_enable">
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_enable">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(5, 144, 110);</string>
           </property>
           <property name="text">
            <string>Géométrie active dans Géonature :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="rdb_yes">
           <property name="text">
            <string>Oui</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="rdb_no">
           <property name="text">
            <string>Non</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pb_additionalFilter">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(5, 144, 110);
</string>
      </property>
      <property name="text">
       <string>  Filtre - Données additionnelles</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/plugins/Geonature-GIS-main/icons/plus.png</normaloff>:/plugins/Geonature-GIS-main/icons/plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="lbl_filterparam">
      <property name="text">
       <string>**Affichage des paramètres du filtre additionnel s'il y a lieu**</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pb_runquery">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>  EXÉCUTER</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/plugins/Geonature-GIS-main/icons/executer_traitement.png</normaloff>:/plugins/Geonature-GIS-main/icons/executer_traitement.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_typegeom">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(5, 144, 110);</string>
      </property>
      <property name="text">
       <string>Types de géométries trouvées : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="lw_typegeomresult">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Point (51)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Polygone (1)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pb_loadlayer">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(5, 144, 110);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string> Charger la couche dans QGIS</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/plugins/Geonature-GIS-main/icons/view.png</normaloff>:/plugins/Geonature-GIS-main/icons/view.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_bt">
      <property name="windowModality">
       <enum>Qt::NonModal</enum>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>20</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>2</width>
        <height>2</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 127);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="GL_export">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="2" column="2">
       <widget class="QPushButton" name="pb_export">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>EXPORTER</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/plugins/Geonature-GIS-main/icons/export_w.png</normaloff>:/plugins/Geonature-GIS-main/icons/export_w.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_folder">
        <property name="text">
         <string>Dossier</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_expformat">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cb_expformat">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>GeoPackage</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GeoJSON</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CSV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XLSX</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsFileWidget" name="qfw_folder"/>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="pb_quit">
      <property name="minimumSize">
       <size>
        <width>95</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>qgscheckablecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
